client/utils/charge/index.ts

```diff
@@ -120,3 +121,15 @@ export const getChargeAmounts = ( charge: Charge ): ChargeAmounts => {
 
 	return balance;
 };
+
+export const getChargeChannel = ( type: string ): string => {
+	if (
+		type === 'card_present' ||
```

Comment:
A small detail here. This method is called in two places: The individual transaction page, and the list of transactions page.

In the individual transaction page, we are getting the data from Stripe, so we pass the `transaction.type` which can be `card_present` or `interac_present` for In-Person payments.

In the list of transactions, the `type` holds the brand of the payment method, so we aren't passing it. Instead, we pass the `transaction.channel` directly, which might be `in_person|online`. The argument is called type, which might feel incorrect in this case, so just wanted to point it out.